.travis/docker-compose-travis.yml

```diff
@@ -0,0 +1,17 @@
+# This is only currently used by Travis for testing
+version: "2"
+
+services:
+  elasticsearch:
+    image: edxops/elasticsearch:latest
```

Comment:
I imagine this is testing against the upgrade?  It probably won't be an issue if we don't have the older image, but is the plan to eventually supply a version?  Should we do that now?